src/common/classes/FpeControl.h

```diff
@@ -223,7 +223,7 @@ inline bool isinf(double x)
 	return (!_finite (x) && !isnan(x));
 }
 #elif !defined(DARWIN)
-#ifndef isinf
+#if !defined (isinf) && !defined(_GLIBCXX_CMATH)
```

Comment:
It's fine for me to merge in regard to the github action. Let's see @AlexPeshkoff about this line change.

Reply:
We already have over-complicated isinf() in FB3, with suggested here change it becomes even worse. IMO that's not good. Please check https://github.com/FirebirdSQL/firebird/tree/isinf - it should build on alpine w/o problems. After your confirmation I will merge that branch into main tree, and you will be able to add github action for alpine.

Reply:
@AlexPeshkoff Everything is ok